src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-hivemq.mdx

```diff
@@ -0,0 +1,12701 @@
+</Callout>
+
+Complete the steps below to collect metrics from HiveMQ:
+
+<Steps>
+
+  <Step>
+    ## Sign up for New Relic! [#signup]
+
+If you haven't already done so, sign up for a free [New Relic account](https://newrelic.com/signup) and get the <LicenseKey/>for the New Relic account to which you want to report data.
+    </Step>
+
+  <Step>
+    ## Review the prerequisites [#prerequisites]
+
+[Configure the prometheus extension](https://github.com/hivemq/hivemq-prometheus-extension/blob/master/README.adoc) in your HiveMQ message platform.
+    </Step> 
+
+   <Step>
+  ## Install the Opentelemetry collector [#install-opentelemetry-collector]
+
+Download and install the Opentelemetry collector following [Opentelemetry docs](https://opentelemetry.io/docs/collector/getting-started/).
+
+    </Step>
+
+   <Step>
+    ## Configure the Opentelemetry collector [#configure-opentelemetry-collector]
+
+Create a new file called `config.yaml` from the example below.
+
+Replace the following keys in the file with your own values:
+
+* HiveMQ Prometheus extension:
+    * HIVEMQ_PROMETHEUS_HOSTNAME
+    * HIVEMQ_PROMETHEUS_PORT
+    * HIVEMQ_PROMETHEUS_METRICS_PATH
+* [New Relic License key](/docs/apis/intro-apis/new-relic-api-keys/#license-key):
+	* NEW_RELIC_LICENSE_KEY
+
+```yaml
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: "hive-mq"
+          static_configs:
+            - targets: ["HIVEMQ_PROMETHEUS_HOSTNAME:HIVEMQ_PROMETHEUS_PORT"]
+          metrics_path: HIVEMQ_PROMETHEUS_METRICS_PATH
+
+exporters:
+  otlp:
+    endpoint: https://otlp.nr-data.net:4317
+    headers:
+      api-key: NEW_RELIC_LICENSE_KEY
+processors:
+  batch:
+  memory_limiter:
+    limit_mib: 400
+    spike_limit_mib: 100
+    check_interval: 5s
+service:
+  telemetry:
+    logs:
+  pipelines:
+    metrics:
+      receivers: [prometheus]
+      processors: [batch, memory_limiter]
+      exporters: [otlp]
+```
+
+You may check [Opentelemetry Collector docs](https://opentelemetry.io/docs/collector/configuration/) and [Prometheus configuration docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/) for additional configuration details.
+    </Step>
+
+   <Step>
+  ## Run the OpenTelemetry collector [#run-collector]
+
+Run the Opentelemetry collector (the way to run it may vary depending on the choosen installation method). Example:
+
+```
+/usr/bin/otelcol --config ./config.yaml
+```
+    </Step>
+
+</Steps>
+
+## Explore your data
+
+You may use the [metrics explorer](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer) to check the metrics being ingested. All metrics reported by HiveMQ prometheus extension start with `com_hivemq_` prefix.
+
+## Metric data details [#hivemq-metrics]
+
+The table below describes the kinds of metrics we collect. 
```

Comment:
@sigilioso Is it correct to say we collect these metric types?

Reply:
Well, these metrics are an example because the metrics being collected may be different depending on the user's hive-mq setup (as explained In the _callout_ below). Could we that these metrics are an example?